### PR TITLE
[Docs] fix failure of python snippet

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - regression was introduced by https://github.com/openvinotoolkit/openvino/pull/20294

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by mryzhov — APPROVED** [review]

File: docs/snippets/ov_auto.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 import openvino.properties.device as device
 import openvino.properties.hint as hints
 import openvino.properties.streams as streams
-import properties.enable_profiling as enable_profiling
```
Comment (yangwang201911) [inline docs/snippets/ov_auto.py:11]: maybe we can correct it as below code.
`import openvino.properties.enable_profiling as enable_profiling`

File: docs/snippets/ov_auto.py [hunk 2]
```diff
@@ -167,7 +166,7 @@ def part5():
     cpu_config = {
         hints.performance_mode: hints.PerformanceMode.LATENCY,
         streams.num: 8,
-        enable_profiling: True
+        properties.enable_profiling: True
```
Comment (yangwang201911) [inline docs/snippets/ov_auto.py:169]:
```suggestion
        enable_profiling: True
```

Comment (yangwang201911) [reply docs/snippets/ov_auto.py:169]: we are fixing this issue in another PR https://github.com/openvinotoolkit/openvino/pull/20531
please ignore above comments.

**Comment by yangwang201911** [comment]
we are fixing this issue in another PR https://github.com/openvinotoolkit/openvino/pull/20531
please ignore above comments.

File: .github/workflows/linux.yml [hunk 3]
```diff
@@ -5,17 +5,19 @@ on:
...
-      - '**/docs/**'
-      - 'docs/**'
-      - '**/**.md'
-      - '**.md'
+    paths:
+      - 'docs/snippets/**'
+      - '!**/docs/**'
+      - '!docs/**'
+      - '!**/**.md'
+      - '!**.md'
```
Comment (mryzhov) [inline .github/workflows/linux.yml:13]:
```suggestion
    paths:
      - '**'
      - '!**/docs/**'
      - '!docs/**'
      - '!**/**.md'
      - '!**.md'
      - 'docs/snippets/**'
```

File: .github/workflows/linux.yml [hunk 4]
```diff
@@ -5,17 +5,19 @@ on:
...
```
Comment (mryzhov) [inline .github/workflows/linux.yml:20]:
```suggestion
    paths:
      - '**'
      - '!docs/**'
      - '!**/**.md'
      - '!**.md'
      - '!**/docs/**'
      - 'docs/snippets/**'
```

**Comment by mryzhov** [comment]
> we are fixing this issue in another PR #20531 please ignore above comments.

it didn't fix the issue

File: .github/workflows/linux.yml [hunk 5]
```diff
@@ -5,17 +5,19 @@ on:
     - cron: '0 0 * * 3,6'
   workflow_dispatch:
   pull_request:
-    paths-ignore:
-      - '**/docs/**'
-      - 'docs/**'
-      - '**/**.md'
-      - '**.md'
+    paths:
+      - 'docs/snippets/**'
```
[the remark this answers is not in the document]
Comment (akuporos) [reply .github/workflows/linux.yml]: updated

**Review by jiwaszki — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**